ckanharvester.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+from sqlalchemy import create_engine
+from sqlalchemy import MetaData
+from ckanext.harvest.model import HarvestObject
+import logging
+from base import HarvesterBase
+
+log = logging.getLogger(__name__)
+
+
+class CKANHarvester(HarvesterBase):
+    def _connect(self, user, password, db, host='192.168.56.101'):
```

Comment:
`host` default value should not be hardcoded to a specific local network address.